CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## New contributor guide
+
+To get an overview of the project, read the [README](README.md). Here are some resources to help you get started with open source contributions:
+
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+
+## Getting started
+
+Bytewax is a Python library backed by Rust code. The codebase is split between Python code in [`pysrc/bytewax`](https://github.com/bytewax/bytewax/tree/main/pysrc/bytewax) and Rust code in [`src`](https://github.com/bytewax/bytewax/tree/main/src). PyO3 is used as the interface between the two. Bytewax operators and the execution of dataflows are handled at the Rust layer, the inputs and outputs can be written in either Rust or in Python.
+
+To develop with Bytewax locally you will need to install Rust and use [Maturin](https://github.com/PyO3/maturin) to build the code into a Python module locally. 
+
+### Issues
+
+#### Create a new issue
+
+If you spot a problem with Bytewax, search if an issue already exists. If a related issue doesn't exist, you can open a new issue using a relevant [issue templat](https://github.com/bytewax/bytewax/issues/new/choose). 
+
+#### Solve an issue
+
+Scan through our [existing issues](https://github.com/bytewax/bytewax/issues) to find one that interests you. You can narrow down the search using `labels` as filters. As a general rule, we don’t assign issues to anyone. If you find an issue to work on, you are welcome to open a PR with a fix.
+
+Important Labels:
+**Good First Issue** - These are a good entry point for contributing to Bytewax.
+**Help Wanted** - These are issues we are looking for the community to help contribute to.
+
+### Make Changes
+
+#### Make changes locally
+
+2. Fork the repository.
+3. Install Rust (check version)
+4. Install Maturin `pip install maturin` ([check version](https://github.com/bytewax/bytewax/blob/main/pyproject.toml))
+5. Install dev requirements `pip install -r dev.requirements.txt`
+5. Run `Maturin Develop`
+6. Test your changes locally.
+7. If possible, add a test that will run with CI.
+
+### Commit your update
+
+Commit the changes once you are happy with them. 
+
+### Pull Request
+
+When you're finished with the changes, create a pull request, also known as a PR.
+- Fill the "Ready for review" template so that we can review your PR. This template helps reviewers understand your changes as well as the purpose of your pull request. 
```

Comment:
```suggestion
- Fill out the "Ready for review" template so that we can review your PR. This template helps reviewers understand your changes as well as the purpose of your pull request. 
```

Reply:
Do we have a link to this template?

Reply:
Incorrectly used template when it is a label.